input/otlp/traces.go

```diff
@@ -257,11 +263,11 @@ func TranslateTransaction(
 			setLabel(k, event, ifaceAttributeValue(v))
 		case pcommon.ValueTypeInt:
 			switch kDots {
-			case semconv.AttributeHTTPStatusCode:
+			case semconv.AttributeHTTPStatusCode, attributeHttpResponseStatusCode:
 				isHTTP = true
 				httpResponse.StatusCode = uint32(v.Int())
 				http.Response = &httpResponse
-			case semconv.AttributeNetPeerPort:
+			case semconv.AttributeNetPeerPort, attributeServerPort:
```

Comment:
We assume that transactions represent HTTP server operations, so should follow https://opentelemetry.io/docs/specs/otel/trace/semantic_conventions/http/#http-server-semantic-conventions

Note that we are setting the `event.source.port` field from this value. I don't think we want to set `event.source.port = server.port` -- `event.source.port` is the port of the client.